BraveRewardsUI/Welcome/WelcomeViewController.swift

```diff
@@ -67,4 +71,17 @@ class WelcomeViewController: UIViewController {
       })
     }
   }
+  
+  private func tappedDisclaimerLink(_ url: URL) {
```

Comment:
this `tappedDisclaimerLink` method looks the same in 3 view controllers, perhaps we could share it in one place?
Idk, maybe a protocol for that would be good, maybe it's an overkill